R/pkg/R/functions.R

```diff
@@ -345,6 +345,17 @@ NULL
 #' head(tmp)}
 NULL
 
+#' ML functions for Column operations
+#'
+#' ML functions defined for \code{Column}.
+#'
+#' @param x Column to compute on.
+#' @param ... additional argument(s).
+#' @name column_ml_functions
+#' @rdname column_ml_functions
+#' @family ml functions
```

Comment:
Can we add an example ( `dontrun`) while we're making a followup to address https://github.com/apache/spark/pull/29917/files#r499340542?